PythonVisualizations/SimpleQueue.py

```diff
@@ -271,6 +314,12 @@ def clickInsertRear(self):
             self.insertRear(entered_text)
             self.clearArgument()
 
+    def clickPeek(self):
+        val = self.peek()
+        
+        if val: self.setMessage("Value {} is at the front of the queue!".format(val))
+        else: self.setMessage("Error! Queue is empty.")  
```

Comment:
I think we should reserve the use of the exclamation mark for error messages or something amazing done by the user (like solving the Tower of Hanoi).  It's OK to use it in line 321 for the error, but it should not be used in the normal status message of line 320.  It's not a surprise what value is at the front of the queue.

Reply:
I had modified the text from the stack which uses an exclamation point for peek. I don't have a preference either way, but maybe we should change stack if we are removing the exclamation point here.

Reply:
Yes, we should change Stack.py and any others, but it's a minor thing compared to some of the other changes needed.